src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -295,53 +295,52 @@ private Assembly Resolve(AssemblyLoadContext loadContext, AssemblyName assemblyN
         // If there are multiple version of the assembly, load the latest.
         private bool TryFindInGAC(AssemblyName assemblyName, out string assemblyFilePath)
         {
-            bool assemblyFound = false;
             assemblyFilePath = null;
+            if (Internal.InternalTestHooks.DisableGACLoading) { return false; }
```

Comment:
Please spread it over 4 lines for consistency

```C#
if (Internal.InternalTestHooks.DisableGACLoading)
{
    return false;
}
```

Reply:
Done.